### ISSUE TITLE
Import/upload PDFs or EPUBs

### ISSUE THREAD

**Issue by xplosionmind** [wallabag/wallabag#5486]
**Is your feature request related to a problem? Please describe.**
Not related to a bug, but to issue #2229.

**Describe the solution you'd like**
Uploading PDFs or EPUB books to be read; having all the things I need to read in one place, not only what is in the form of a web page.

**Describe alternatives you've considered**
Retrieving the URL for this resource online, if it has ever been uploaded by somebody.

**Additional context**
Let’s say I just downloaded an academic paper and I want to read it on my mobile Wallabag app, and being able to list it among the other things I need to read. Uploading the PDF of the paper to Wallabag satisfies my need.

**Comment by almet** [comment]
That would be great! Did you manage to solve your issue?

**Comment by xplosionmind** [comment]
nope, not yet!  On Wed, Feb 16, 2022 at 10:08 PM, Alexis Metaireau ***@***.***> wrote:  
That would be great! Did you manage to solve your issue?

**Comment by thiswillbeyourgithub** [comment]
I would be incredibly interested by this. That would be a killer feature and it would definitely be worth paying the premium tier account! (Although I already have it :))

**Comment by dgoldstein0** [comment]
+1, I'm considering using wallabag and without pdf support, I'm likely to adopt a different tool.

I tried to add https://www.cse.chalmers.se/~rjmh/Papers/arrows.pdf to my wallabag.it both via wallabagger in chrome (it grabs an empty page) and by directly adding it on the website - the latter which saves garbled text - likely the raw pdf bytes interpreted as ascii or unicode.

It appears some people think this is a bug instead of a feature request: #7550 #7529

... and #116 suggests it's expected to work, but given all the reports any support must be very half-baked.  #2655 also looks like a partial-duplicate of this request.

**Comment by l1mey112** [comment]
This is still a huge problem, doesn't work for me either.
